### DISCUSSION TITLE
USB pendrive

### DISCUSSION THREAD

**Discussion by Mariusz0051** [rern/rAudio#1265] [Bug Reports]
Hello. Sorry for such a simple question.... I installed rAudio on Raspberry Pi 3+. I connected the 3.5 display - it works.
But I have a problem with pendrive, mp3, flac. Nothing songs. The flash drive is mounted.

**Comment by R101G** [comment]
Have you got FFmpeg and Library Auto Update enabled in the Player menu?

**Comment by Mariusz0051** [comment]
Yes.
![image](https://user-images.githubusercontent.com/96250590/224480922-11131774-a0d5-441b-a0b7-86b209fc7887.png)

![image](https://user-images.githubusercontent.com/96250590/224480891-b77c361b-8aee-4a76-a98f-ecad3492e9c5.png)

**Comment by hjfisch** [comment]
Did you try the right circle of Library and "rebuild entire Database"?

**Comment by Mariusz0051** [comment]
Thank you, it works. I have one more question. How can I buy such a card? I see it has a GPI connector. Will a 3.5 inch display and sound card work?

https://botland.com.pl/raspberry-pi-karty-dzwiekowe-i-hifiberry/18129-hifiberry-dac2-pro-karta-dzwiekowa-dla-raspberry-pi-4b3b3b-5904422363000.html

https://botland.com.pl/wyswietlacze-raspberry-pi/4479-ekran-dotykowy-a-rezystancyjny-lcd-tft-35-320x480px-gpio-dla-raspberry-pi-4-3-2-b-zero-waveshare-9904-5904422371296.html
